code-challenges.test.js

```diff
@@ -25,35 +25,168 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+describe("divisibleByThree", () => {
+    it("decides if the number inside the object is divisible by three or not", () => {
+        const object1 = { number: 15 }
+        // Expected output: "15 is divisible by three"
+        const object2 = { number: 0 }
+        // Expected output: "0 is divisible by three"
+        const object3 = { number: -7 }
+        // Expected output: "-7 is not divisible by three"
+        expect(divisibleByThree(object1.number)).toEqual("15 is divisible by three")
+        expect(divisibleByThree(object2.number)).toEqual("0 is divisible by three")
+        expect(divisibleByThree(object3.number)).toEqual("-7 is not divisible by three")
+     
+    })
+})
+
+// FAIL  ./code-challenges.test.js
+// divisibleByThree
+// ✕ decides if the number inside the object is divisible by three or not (1 ms)
+
+// ● divisibleByThree › decides if the number inside the object is divisible by three or not
+
+// ReferenceError: divisibleByThree is not defined
+
 
 // b) Create the function that makes the test pass.
+// PSEUDOCODE
+    /* 
+        -input: an object
+        -output: a string with interpolated object stating whether the object content is divisible by 3 or not
```

Comment:
You could expand a bit more here in this pseudocode. You can do object destructuring, talk about using modulo etc.